beacon_node/lighthouse_network/src/rpc/codec/ssz_snappy.rs

```diff
@@ -419,14 +419,19 @@ fn context_bytes<E: EthSpec>(
                         }
                     };
                 }
-                RPCResponse::BlobsByRange(_)
-                | RPCResponse::BlobsByRoot(_)
-                | RPCResponse::DataColumnsByRoot(_)
-                | RPCResponse::DataColumnsByRange(_) => {
-                    // TODO(das): If DataColumnSidecar is defined as an Electra type, update the
-                    // context bytes to point to ForkName::Electra
+                RPCResponse::BlobsByRange(_) | RPCResponse::BlobsByRoot(_) => {
                     return fork_context.to_context_bytes(ForkName::Deneb);
                 }
+                RPCResponse::DataColumnsByRoot(_) | RPCResponse::DataColumnsByRange(_) => {
+                    // TODO(das): Remove deneb fork after `peerdas-devnet-2`.
+                    return if fork_context.spec.eip7594_fork_epoch
+                        == fork_context.spec.deneb_fork_epoch
```

Comment:
Just confirming if this condition is specifically for devnet testing.
If we start a network with deneb_fork_epoch == electra_fork_epoch == peerdas_fork_epoch = 0, then this will give deneb context bytes even though we probably want electra.

Reply:
Created https://github.com/sigp/lighthouse/pull/6325